django/contrib/admin/static/admin/js/SelectFilter2.js

```diff
@@ -195,4 +195,12 @@ Requires core.js, SelectBox.js and addevent.js.
         }
     };
 
+    addEvent(window, "load", function(e) {
+        $('select.selectfilter, select.selectfilterstacked').each(function() {
+            var $el = $(this),
```

Comment:
I haven't seen this comma style used elsewhere -- is there some reason to prefer it instead of repeating var on each line?